fdbclient/admin_client.go

```diff
@@ -318,13 +318,21 @@ func (client *cliAdminClient) GetExclusions() ([]fdbv1beta2.ProcessAddress, erro
 	lines := strings.Split(output, "\n")
 	exclusions := make([]fdbv1beta2.ProcessAddress, 0, len(lines))
 	for _, line := range lines {
+		line = strings.TrimSpace(line)
+		if len(strings.Split(line, " ")) != 1 {
+			continue
+		}
 		exclusionMatch := exclusionLinePattern.FindStringSubmatch(line)
 		if exclusionMatch != nil {
 			pAddr, err := fdbv1beta2.ParseProcessAddress(exclusionMatch[1])
 			if err != nil {
 				return nil, err
 			}
 			exclusions = append(exclusions, pAddr)
+			continue
+		}
+		if strings.HasPrefix(line, "locality") {
```

Comment:
I think a better way to do is querying the FDB status and returning all excluded processes that are under `excluded_servers` this way we don't need to use a regex (and I would expect that the regex from above actually matches the locality string). Have you tested this locally?

In another step( not your PR) we could query the mgmt API directly.